config/locales/server.en.yml

```diff
@@ -5206,6 +5206,8 @@ en:
         description: "Restore the post so that all users can see it."
       disagree:
         title: "No"
+      discard_post:
+        title: "Discard Post"
```

Comment:
What are the rules of capitalization here?

I see just above we have: `title: "Delete post"`

Reply:
It's all over the place. What I did here is keep it consistent within the same menu, but between menus it's definitely not consistent.